data_steward/deid/config/internal_tables/src_hpos_to_allowed_states.csv

```diff
@@ -1,36 +1,49 @@
 State,value_source_concept_id,src_hpo_id
-PIIState_AL,1585261,saou_uab
-PIIState_AL,1585261,saou_usahs
-PIIState_AZ,1585264,uamc_banner
+PIIState_WI,1585410,aouw_mcri
+PIIState_WI,1585410,aouw_mcw
+PIIState_WI,1585410,aouw_uwh
+PIIState_CT,1585268,chci
+PIIState_TN,1585310,chs
 PIIState_CA,1585266,cpmc_ceders
 PIIState_CA,1585266,cpmc_ucd
 PIIState_CA,1585266,cpmc_uci
 PIIState_CA,1585266,cpmc_ucsd
 PIIState_CA,1585266,cpmc_ucsf
 PIIState_CA,1585266,cpmc_usc
-PIIState_CA,1585266,syhc
-PIIState_CT,1585268,chci
-PIIState_FL,1585272,seec_miami
-PIIState_FL,1585272,seec_ufl
-PIIState_GA,1585273,seec_emory
-PIIState_GA,1585273,seec_morehouse
+PIIState_SC,1585308,ecchc
+PIIState_NY,1585297,hrhc
 PIIState_IL,1585277,ipmc_northshore
 PIIState_IL,1585277,ipmc_nu
 PIIState_IL,1585277,ipmc_rush
 PIIState_IL,1585277,ipmc_uchicago
 PIIState_IL,1585277,ipmc_uic
+PIIState_MS,1585289,jhchc
 PIIState_MA,1585286,nec_bmc
 PIIState_MA,1585286,nec_phs
-PIIState_MI,1585287,tach_hfhs
-PIIState_MI,1585287,trans_am_spectrum
-PIIState_MS,1585289,jhchc
-PIIState_NY,1585297,hrhc
 PIIState_NY,1585297,nyc_cornell
 PIIState_NY,1585297,nyc_cu
 PIIState_NY,1585297,nyc_hh
 PIIState_PA,1585305,pitt
-PIIState_SC,1585308,ecchc
-PIIState_TN,1585310,chs
-PIIState_WI,1585410,aouw_mcri
-PIIState_WI,1585410,aouw_mcw
-PIIState_WI,1585410,aouw_uwh
+PIIState_PA,1585305,pitt_temple
+PIIState_AL,1585261,saou_cgmhs
+PIIState_AL,1585261,saou_hhsys
```

Comment:
This site name is not available in `hpo_id_bucket_name` look_up table. this shouldn't be here as per the acceptance criteria in the ticket.

Reply:
You're right, let me update it.